fashion-example-query/README.md

```diff
@@ -3,83 +3,126 @@
 **Table of Contents**
 
 - [Run the Hello World example using query language](#run-the-hello-world-example-using-query-language)
-  - [🗝️ Key Concepts](#️-key-concepts)
-  - [🏃 Run the Flows](#-run-the-flows)
-  - [Results](#results)
-  - [QueryLanguage](#querylanguage)
+  - [🗝️ Expected knowledge](#Expected-knowledges)
+  - [🔮 Overview of the files](#Overview-of-the-files)
+  - [🏃 Run the Flows](#run-the-flows)
+  - [🌟 Results](#results)
+  - [🧞‍QueryLanguage](#querylanguage)
+  - [💫 Deploy with Docker](#Deploy-with-docker)
+  - [Next steps](#next-steps)
+  - [Community](#community)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# Run the Hello World example using query language
+# Run Fashion-MNIST Example Using Query Language
 
-## 🗝️ Key Concepts
+This is an extension of the Hello World example using QueryLanguage. 
+We will use the FashionMNIST dataset to index and query. The dataset has different categories such as t-shirts, trousers, and pullovers. We will use QueryLanguage to create a different index per each category so we can query through them. 
 
-First of all, read up on [Jina 101](http://101.jina.ai) so you have a clear understanding of how Jina works. We're going to refer to those concepts a lot. We assume you already have some knowledge of Python and machine learning.
 
-This is an extension of the Hello World example, but here we use QueryLanguage to create an index per category and then, with that we can filter by category
+## 🗝️ Expected knowledge 
```

Comment:
I know it looks very self-explanatory, but Google recommends that lists should be introduced.  See here: https://developers.google.com/tech-writing/one/lists-and-tables

Something as simple as: 
`Before running this example, we recommend you meet the following requirements. `